sched/Kconfig

```diff
@@ -1253,6 +1253,7 @@ config SCHED_STARTHOOK
 config SCHED_ATEXIT
 	bool "Enable atexit() API"
 	default n
+	select SCHED_ONEXIT
```

Comment:
Are both SCHED_ATEXIT and SCHED_ONEXIT required?  They seem to be synonyms.

Reply:
Yes, in this patch SCHED_ATEXIT is always depend on SCHED_ONEXIT (with little overhead), as both them enabled on master does.

So maybe we can merge them into one config item.

Reply:
Yes, but we will have to change a few affected defconfig files.